js/script.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+$(function() {
+
+    function checkUser(e, elements, users) {
+        let active = elements.activeName;
+        if(e.keyCode === 40){
+            if(elements.isOpen && active >= 0 && active < 9){
+                elements.activeName = active + 1;
+                }else {
+                elements.activeName = 0;
+                }
+            return makeElemActive(active, elements.activeName)
+            }
+            if(e.keyCode === 38) {
+                if (elements.isOpen && active > 0 && active <= 9) {
+                    elements.activeName = active - 1;
+                    } else {
+                    elements.activeName = 9;
+                    }
+                return makeElemActive(active, elements.activeName);
+            }
```

Comment:
@lianatumikyan  please make the local variable for elements.activeName, you use it more than one time under this code.